membership-attribute-service/app/Global.scala

```diff
@@ -0,0 +1,34 @@
+import controllers.{NoCache, Cached}
+import filters.AddEC2InstanceHeader
+import models.{MembershipAttributes, ApiError, ApiErrors, ApiResponse}
+import play.api.Logger
+import play.api.mvc.{RequestHeader, Result, WithFilters}
+import play.filters.csrf._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+object Global extends WithFilters(CSRFFilter(), AddEC2InstanceHeader) {
+
+  private val logger = Logger(this.getClass)
```

Comment:
Across the other projects we use a `Logging` trait, which essentially just does this (e.g. https://github.com/guardian/identity-federation-api/blob/master/src/main/scala/com/gu/identity/oauth/util/Logging.scala)

For this simple case it makes no difference, but for Federation we ended up with some more advanced log message massaging, so it made sense to put it all in one place.